test/download-handler.js

```diff
@@ -35,7 +35,7 @@ exports.tests = {
         ShellJS.rm(tmpPath);
 
         // Download
-        var url = "http://www.linuxfoundation.org";
+        var url = "https://www.linuxfoundation.org";
```

Comment:
This is a test for the http protocol (see test name httpStream). I would recommend to use another site that is using plain http. The corresponding https test is further below. Make sure that the test site does not use redirections.